CONTRIBUTING.md

```diff
@@ -47,47 +47,47 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+the extension out with. It is not an exhaustive showcase of DVC's features, so
+testers are encouraged to try other DVC repositories -- especially real-world
+cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Run `yarn setup:venv` from the
+  root of this repo to set up a virtual Python environment for the demo project
+  (in `demo/.env`).
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Important: Open the `demo/` folder of this repo directly in VS Code.
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Pull the project data with the extension (either from the **DVC Tracked**
+  panel in the _File Explorer_ or the **DVC panel** in the _Source Code_ view)
+  or by running `dvc pull` from a VS Code _Terminal_.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+- In order to [run experiments] in the demo project, the virtual env should be
+  loaded through the Microsoft [Python extension] (please install it), having
+  this in `.vscode/settings.json`:
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+  ```json
+  "python.defaultInterpreterPath": "demo/.env/bin/dvc"
```

Comment:
[I] This is the option for manually specifying a [Python interpreter](https://code.visualstudio.com/docs/python/environments#_manually-specify-an-interpreter) for the Python extension. Why would you point it at the demo project's version of DVC?

Reply:
Typo, should be `python`. Fixing